data/vibe/data/serialization.d

```diff
@@ -571,6 +579,67 @@ private template serializeValueImpl(Serializer, alias Policy) {
 	}
 }
 
+///
+package template isSerializerSupportSinkType (SerT, ObjT)
+{
+	enum isSerializerSupportSinkType = is(typeof(SerT.serializeSinkType!(ObjT)(ObjT.init)) == void);
+}
+
+///
+package template isObjectSupportSinkType (ObjectT)
+{
+	enum isObjectSupportSinkType = is(typeof(ObjectT.toString(cast(SerializeSinkDg) null)) == void) &&
```

Comment:
```suggestion
	enum isObjectSupportSinkType = is(typeof(ObjectT.toString(SerializeSinkDg.init))) &&
```

`X.init` is a neat way to get a default value typed as `X`. Remember, every cast is an aberration ;)
And I don't think we need to force the return type to be `void`. Why would we ?